packages/backend/src/rates/service.ts

```diff
@@ -107,13 +112,26 @@ class RatesServiceImpl implements RatesService {
     const url = this.deps.pricesUrl
     if (!url) return {}
 
-    const res = await this.axios.get(url).catch((err) => {
+    const res = await this.axios.get<PricesResponse>(url).catch((err) => {
       this.deps.logger.warn({ err: err.message }, 'price request error')
       throw err
     })
-    this.pricesRequest = Promise.resolve(res.data)
+
+    const { base, rates } = res.data
+    if (!base) {
+      const errorMessage = 'Base currency was not provided'
+      this.deps.logger.warn({ errorMessage })
+      throw new Error(errorMessage)
+    }
+
+    const data = {
+      [base]: 1.0,
+      ...(rates ? rates : {})
+    }
```

Comment:
In order to make same asset transfers, the base asset needs to be added to the `Prices` object that `loadNow` is returning.